crates/uv-cli/src/lib.rs

```diff
@@ -2606,16 +2606,23 @@ pub struct VenvArgs {
     #[arg(long, value_parser = clap::builder::BoolishValueParser::new(), env = EnvVars::UV_VENV_SEED)]
     pub seed: bool,
 
+    /// Remove any existing files or directories at the target path.
+    ///
+    /// By default, `uv venv` will exit with an error if the given path is non-empty. The
+    /// `--clear` option will instead clear a non-empty path before creating a new virtual
+    /// environment.
+    #[clap(long, short, conflicts_with = "allow_existing", value_parser = clap::builder::BoolishValueParser::new(), env = EnvVars::UV_VENV_CLEAR)]
```

Comment:
Why does this conflict with `allow_existing` rather than override?

Reply:
I was being conservative. I think we'd want `--allow-existing` to override if we took that approach? The presence of the flag might indicate they expect those files to be retained

Reply:
I think you'd probably just have them both override each other?

Reply:
Done